### PR TITLE
Update Howto: send a diagnostic report

### PR DESCRIPTION
**Pull request by MakaraSok**
The instructions and images have been updated based on Keyman 13.

### CODE REVIEW

File: knowledge-base/kb0040.md [hunk 1]
```diff
@@ -12,17 +10,20 @@
...
+    <ol type="1">
+      <li><p>Click on the <span class="guiicon">Keyman Desktop</span> icon <span class="inlinemediaobject"><img src="desktop_images/icon-keyman.png"></span>, on the Windows Taskbar near the clock.</p></li>
+      <li><p>From the Keyman Desktop menu, select <span class="guilabel">Configuration…</span>.</p></li>
+    </ol></div>
 </li><br/>
 
 <li>
   <b>Select the Support tab.</b><br/>
-  <p><img src='assets/kb0040/step2.png' alt='Click Support' /></p>
+  <p><img src="desktop_images/tab-support.png" alt='Click Support' /></p>
```
Comment (darcywong00) [inline knowledge-base/kb0040.md:22]: I'm not seeing this image

File: knowledge-base/kb0040.md [hunk 2]
```diff
@@ -12,17 +10,20 @@
 <ol>
 <li>
   <b>Open Keyman Configuration.</b><br/>
-  <p><img src='assets/kb0040/step1.png' alt='Open Keyman Configuration' /></p>
+  <div class="orderedlist">
+    <ol type="1">
+      <li><p>Click on the <span class="guiicon">Keyman Desktop</span> icon <span class="inlinemediaobject"><img src="desktop_images/icon-keyman.png"></span>, on the Windows Taskbar near the clock.</p></li>
```
Comment (darcywong00) [inline knowledge-base/kb0040.md:15]: I'm not seeing this image

File: knowledge-base/kb0040.md (hunk 1 above)
Comment (MakaraSok) [reply knowledge-base/kb0040.md:22]: I've utilized the existing img found here: https://help.keyman.com/products/desktop/current-version/docs/basic_support_tab.

File: knowledge-base/kb0040.md (hunk 2 above)
Comment (MakaraSok) [reply knowledge-base/kb0040.md:15]: The same as above.

Comment (darcywong00) [reply knowledge-base/kb0040.md:15]: The src should be 
```
<img src="/products/desktop/current-version/docs/desktop_images/icon-keyman.png">
```

File: knowledge-base/kb0040.md (hunk 1 above)
Comment (darcywong00) [reply knowledge-base/kb0040.md:22]: ah, the src should be
`Click Support`

**Pull request merged**